packages/components/src/components/notification-toast/notification-toast.css

```diff
@@ -12,6 +12,7 @@
 :host {
   /* toast */
   --width: 367px;
+  --width-text-container: 271px;
```

Comment:
this is quite a magic number - why exactly 271?

Reply:
The magic number arises as a difference. So as a result of this: 
367px (total width) - 48px (icon-container) - 48px (close-button spacing equal to icon-container) = 271px 

But the question is whether the spacing on the right should always be there or only when a close button is present.

Reply:
then why 367? 👻 you could use [calc](https://developer.mozilla.org/en-US/docs/Web/CSS/calc) to make the value explicit here.

> But the question is whether the spacing on the right should always be there or only when a close button is present.

i'd say only when the button is present

Reply:
367px was the specification from design side 🫠. Added calculation instead of magic number.

Toasts must have a close button ... sorry my fault 🤦‍♂️